src/App.ts

```diff
@@ -10,17 +10,33 @@ const error = debug('team-link:error');
 
 export default class App {
     private app: express.Express;
-    private server!: Server;
+    private port: string;
+    private host: string;
+    private staticPath: string;
+    private clientPath: string;
+    private server: Server;
 
-    constructor () {
+    constructor ({ port = config.port, host = config.host, staticPath = config.staticPath, clientPath = config.clientPath } = {}) {
```

Comment:
Please make the constructor private and enable the `@typescript-eslint/explicit-member-accessibility` rule.
Also, do we really need to dereference each option to a different property? I guess our code will look cleaner if just save all of them as an `options` object.

Reply:
The `@typescript-eslint/explicit-module-boundary-types` looks very promising too.